src/cli/commands/identity/status.ts

```diff
@@ -0,0 +1,40 @@
+import { DateTime } from "luxon";
+import { CommandModule } from "yargs";
+import { BaseCommand, ConfigFileOptions, configOptionBuilder } from "../../BaseCommand";
+
+export const identityStatusHandler = async ({ config }: ConfigFileOptions): Promise<void> => {
+    await new IdentityStatus().run(config);
+};
+export const yargsIdentityStatusCommand: CommandModule<{}, ConfigFileOptions> = {
+    command: "status",
+    describe: "show the status of the identity",
+    handler: identityStatusHandler,
+    builder: configOptionBuilder
+};
+
+export class IdentityStatus extends BaseCommand {
```

Comment:
Should IdentityStatus only refer to the IdentityDeletionProcessStatus? If so, I would expect another name.

Reply:
Na this will show some more info like the identity string and if there also show the identity deletion